### PR TITLE
Adding support for Logback-access events

### PR DESCRIPTION
**Pull request by renanstuchi**
Hello,

Would like to add support for IAccessEvents in this code.

### CODE REVIEW

**Comment by epoh-r7** [comment]
Hi @renanstuchi , could you also add some unit tests for the LogentriesAccessAppender

File: pom.xml [hunk 1]
```diff
@@ -6,10 +6,12 @@
 
         <jackson.version>2.15.0</jackson.version>
         <log4j2.version>2.20.0</log4j2.version>
+        <logback.version>1.3.8</logback.version>
```
Comment (jcolgan-r7) [inline pom.xml:9]: Can we use 1.4.8?

**Review by jcolgan-r7 — COMMENTED** [review]
Looks good overall. Can we format the changes back to having 4-space indentation?

File: pom.xml (hunk 1 above)
Comment (epoh-r7) [reply pom.xml:9]: Nope, would need to bump to java11 for that

**Review by jcolgan-r7 — APPROVED** [review]

**Comment by epoh-r7** [comment]
@renanstuchi thanks for the contribution!

this should be released as 4.0.1 https://github.com/rapid7/r7insight_java/releases/tag/4.0.1

**Comment by renanstuchi** [comment]
Glad to help ! ( sorry just seen this thread now )

**Pull request merged**